yocto/smoke-test-images.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2015 Pelagicore AB
+#      
+# Permission to use, copy, modify, and/or distribute this software for 
+# any purpose with or without fee is hereby granted, provided that the 
+# above copyright notice and this permission notice appear in all copies. 
+#  
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL 
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED  
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR 
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES 
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, 
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, 
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS 
+# SOFTWARE.
+#   
+# For further information see LICENSE
+# 
+#
+# Usage: smoke-test-images.sh <yoctodir> <images>
+#
+# Source the environment and smoke test the supplied images in the yoctodir
```

Comment:
It would be nice to have it mentioned in the description that this is a qemu specific script.

Reply:
Better?